### PR TITLE
Add Author field to the attachment

### LINKED ISSUE #14: Include the user who triggered an event
My use case would be, to see the user who triggered an event. As an example on a build I'd love to see the username similar to how Travis CI handles its Slack Notifications.

```
Build #6105 (1187460) of evil_corp/domination by Evan Sparkman passed in 55 min 50 sec
```

### PR DESCRIPTION
**Pull request by torbjornvatn**
## Purpose
Add a new field to the Slack attachment with the Author/Actor from the github event triggering the action

Fixes #14 

## Changelog

- [x] I have updated `CHANGELOG.md` under "Unreleased" with the changes included in this PR.

### CODE REVIEW

**Comment by triralph** [comment]
I'll be watching this one, I actually want the author on the slack messages as well.

**Comment by ThomasEg** [comment]
...this is nice...

**Comment by ThomasEg** [comment]
Any progress on getting this merged? I've got a couple of requests on this exact feature....

**Comment by thiago-scherrer** [comment]
Hello! Any news about this PR? This feature interests me :)

**Comment by pam-** [comment]
@ThomasEg @thiago-scherrer hello! we've had some changes on the team hence the slow response. We will get to this asap! Thanks for bearing with us.

**Comment by kimegede** [comment]
I would like author as well :)
If we get the conflicts fixed, would it be possible to get this merged soon? 🙏

**Comment by kimegede** [comment]
@torbjornvatn could you please resolve the conflicts? :)
Then hopefully it can speed up the process.

**Comment by torbjornvatn** [comment]
@kimegede fixed it now

**Comment by kimegede** [comment]
Really nice @torbjornvatn! Big high five 🙏
@pam- 😊

File: __tests__/utils.spec.js [hunk 1]
```diff
@@ -30,6 +32,16 @@ describe('Utils', () => {
...
 
+    it('show author/actor', () => {
+      const attachments = buildSlackAttachments({ status: 'STARTED', color: 'good', github: GITHUB_PUSH_EVENT });
+
+      expect(attachments[0].fields.find(a => a.title === 'Author')).toEqual({
+        title: 'Author',
+        value: 'Codertocat',
+        short: true,
+      });
+    });
```
Comment (thiago-scherrer) [inline __tests__/utils.spec.js:43]: 🚀 🚀 🚀 🚀 🚀

**Review by thiago-scherrer — APPROVED** [review]
LGTM 🚀

**Comment by kimegede** [comment]
@pam- can we soon get this one merged please? :)

**Review by kimegede — APPROVED** [review]

**Comment by kimegede** [comment]
Is there maybe someone else we will need to tag, to get included and being able to move forward with this?
(Rather want the source to be improved, than fork it to add this).

**Comment by kimegede** [comment]
Anyone else from Voxmedia there can move this PR forward?
Sorry for tagging that many...
@ealcantara22 @lovitt @luciovilla @maria-mata @nicolezhu @okor @sixfoot6 @stephenmckinney @thomsbg

**Comment by pam-** [comment]
@kimegede We are running with less engineers than usual and might not be able to maintain this for a while. I will confirm that merging this doesn't interfere with any of our repos before pushing the button.

For future features, I encourage you to fork this and make it your own. Thanks!

**Comment by kimegede** [comment]
Thanks for the feedback Pam, that's understood.
Have a good one :)

**Pull request closed without merging**